test/jest/roktManager.spec.ts

```diff
@@ -17,17 +19,85 @@ describe('RoktManager', () => {
         });
     });
 
+    describe('#init', () => {
+        it('should initialize the manager with defaults when no config is provided', () => {
+            roktManager.init({} as SDKInitConfig);
+            expect(roktManager['launcher']).toBeNull();
+            expect(roktManager['filters']).toEqual({
+                userAttributeFilters: undefined,
+                filterUserAttributes: expect.any(Function),
+            });
+            expect(roktManager['filteredUser']).toBeUndefined();
+            expect(roktManager['kit']).toBeNull();
+        });
+        it('should initialize the manager with user attribute filters from a config', () => {
+            const kitConfig: Partial<IKitConfigs> = {
+                name: 'Rokt',
+                moduleId: 181,
+                userAttributeFilters: [816506310, 1463937872, 36300687],
+            };
+
+            const config: SDKInitConfig = {
+                kitConfigs: [kitConfig as IKitConfigs],
+            };
+
+            roktManager.init(config);
+            expect(roktManager['filters']).toEqual({
+                userAttributeFilters: [816506310, 1463937872, 36300687],
+                filterUserAttributes: expect.any(Function),
+            });
+        });
+    });
+
+    describe('#parseConfig', () => {
+        it('should ONLY return the kit config for Rokt', () => {
+            const kitConfig: Partial<IKitConfigs> = {
```

Comment:
for clarity, rename this constant
```suggestion
            const roktKitConfig: Partial<IKitConfigs> = {
```